sklearn/linear_model/setup.py

```diff
@@ -32,6 +30,7 @@ def configuration(parent_package='', top_path=None):
 
         with open(sag_cython_file, "r") as f:
             tmpl = f.read()
+        from Cython import Tempita
```

Comment:
To avoid pep8 related warnings:

```suggestion
        from Cython import Tempita  # noqa
```

Reply:
Hmm, interesting that I can push to a PR but not commit a suggestion.

Reply:
I have amended with the ``  # noqa``, as suggested.

Reply:
@rth If it helps, the 'Commit suggestion' button is grayed out for me as well.